miss_hit.cfg

```diff
@@ -0,0 +1,44 @@
+# Style guide configuration for MISS_HIT
+# Reference: https://florianschanda.github.io/miss_hit/style_checker.html
+
+# Maximum allowed length for a single line of code (in characters)
+# Helps maintain code readability without unnecessary horizontal scrolling.
```

Comment:
Probably here we want to align on what is going to be the convention as in the current version is still erroring:

Reply:
```
Checking code formatting in ./../docs/ ./../test/...
poetry run mh_style ./../docs/ ./../test/
In ../docs/src/bringONSASmFilesToONSASdocs.m, line 1
| % md function used to conver .m files in the ONSAS repo to .md files in this repo
| ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ style: No copyright notice found in docstring [copyright_notice]
In ../docs/src/bringONSASmFilesToONSASdocs.m, line 6
| function bringONSASmFilesToONSASdocs
|          ^^^^^^^^^^^^^^^^^^^^^^^^^^^ style: violates naming scheme for function [naming_functions]
In ../docs/src/m2md.m, line 1
| function m2md(fileIn, fileOut, includeCodeBoolean, iniLine)
|          ^^^^ style: violates naming scheme for function [naming_functions]
In ../docs/src/m2md.m, line 1
| function m2md(fileIn, fileOut, includeCodeBoolean, iniLine)
|          ^^^^ style: Could not find any copyright notice [copyright_notice]
In ../test/gaussIntegrationTest.m, line 1
| % ========================================================================================
| ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ style: No copyright notice found in docstring [copyright_notice]
In ../test/gaussIntegrationTest.m, line 4
| function resultBoolean = gaussIntegrationTest()
|                          ^^^^^^^^^^^^^^^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/gaussIntegrationTest.m, line 56
| function ys = test_fun_to_integrate(xinput)
|               ^^^^^^^^^^^^^^^^^^^^^ style: violates naming scheme for function [naming_functions]
../test/runTestProblems_local.m: style: violates naming scheme for scripts [naming_scripts]
In ../test/runTestProblems_moxunit_disp.m, line 1
| % function for testing ONSAS using moxunit
| ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ style: No copyright notice found in docstring [copyright_notice]
In ../test/runTestProblems_moxunit_disp.m, line 3
| function test_suite = runTestProblems_moxunit_disp
|                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 16
| function test_1
|          ^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 20
| function test_2
|          ^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 24
| function test_3
|          ^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 28
| function test_4
|          ^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 32
| function test_5
|          ^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 36
| function test_6
|          ^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 40
| function test_7
|          ^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 44
| function test_8
|          ^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 48
| function test_9
|          ^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 52
| function test_10
|          ^^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 56
| function test_11
|          ^^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 60
| function test_12
|          ^^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 64
| function test_13
|          ^^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 68
| function test_14
|          ^^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 72
| function test_15
|          ^^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 76
| function test_16
|          ^^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 80
| function test_17
|          ^^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 84
| function test_18
|          ^^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 88
| function test_19
|          ^^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 92
| function test_20
|          ^^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 96
| function test_21
|          ^^^^^^^ style: violates naming scheme for function [naming_functions]
In ../test/runTestProblems_moxunit_disp.m, line 100
| function test_22
|          ^^^^^^^ style: violates naming scheme for function [naming_functions]
MISS_HIT Style Summary: 5 file(s) analysed, 32 style issue(s)
make: *** [Makefile:26: format_check] Error 1
```

Reply:
Yes, however this could be solved in another PR? maybe this pr adds miss_hit and in another we define the format...

Reply:
Makes sense, we can do that. Merge the machinery and then we can decide, I was thinking the automation of pre-commit can be added in a different PR after we clean all formats issues.

Reply:
I've tried to describe the plan https://github.com/ONSAS/ONSAS/issues/677#issue-2010341755, feel free to edit or modify as you consider the best to do a safe and sound auto formatting migration.